app/models/uphold_connection.rb

```diff
@@ -406,6 +408,17 @@ def find_or_create_uphold_card!
     end
   end
 
+  def has_deposit_capability?
+    result = connection_client.users.get_capability("deposits")
+
+    case result
+    when UpholdUserCapability
+      result.requirements.empty? && result.restrictions.empty?
+    else
+      false
```

Comment:
This actually isn't the best way to do this.  I should really throw another exception here because this would only happen if the request failed or something.